.github/workflows/dhis2-verify-app.yml

```diff
@@ -0,0 +1,153 @@
+name: 'dhis2: verify (app)'
+
+# Requirements:
+#
+# - Org secrets:
+#       DHIS2_BOT_GITHUB_TOKEN
+#       DHIS2_BOT_APPHUB_TOKEN
+# - Enable the 'jobs.e2e' section if applicable
+# - Customize the properties if needed:
+#       'jobs.release.steps.*.cwd'
+#       'jobs.release.steps.*.build-dir'
+
+on: push
+
+concurrency:
+    group: ${{ github.workflow}}-${{ github.ref }}
+
+env:
+    GIT_AUTHOR_NAME: '@dhis2-bot'
+    GIT_AUTHOR_EMAIL: 'apps@dhis2.org'
+    GIT_COMMITTER_NAME: '@dhis2-bot'
+    GIT_COMMITTER_EMAIL: 'apps@dhis2.org'
+    D2_VERBOSE: true
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: c-hive/gha-yarn-cache@v1
+            - run: yarn install --frozen-lockfile
+
+            - name: Build
+              run: yarn d2-app-scripts build
+
+            - uses: actions/upload-artifact@v2
+              with:
+                  name: app-build
+                  path: |
+                      **/build
+                      !**/node_modules
+                  retention-days: 1
+
+    lint:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: c-hive/gha-yarn-cache@v1
+            - run: yarn install --frozen-lockfile
+
+            # Can be removed if translations aren't required for tests,
+            # or if not using the App Platform
+            - name: Generate translations
+              run: yarn d2-app-scripts i18n generate
+
+            - name: Lint
+              run: yarn d2-style check
+
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: c-hive/gha-yarn-cache@v1
+            - run: yarn install --frozen-lockfile
+
+            # Can be removed if translations aren't required for tests,
+            # or if not using the App Platform
+            - name: Generate translations
+              run: yarn d2-app-scripts i18n generate
+
+            - name: Test
+              run: yarn d2-app-scripts test
+
+    e2e:
+        runs-on: ubuntu-latest
+        if: "!github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
+
+        strategy:
+            fail-fast: false
+            matrix:
+                containers: [1, 2, 3, 4]
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
```

Comment:
We should do minimum 14.x and preferably 16.x

Reply:
Bumped to 14.x